charts/dvpe-deployment-gloo/CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
 
+## [4.0.0]
+
+### Changed
+* Secrets generation for new ExternalSecret
+
+  Parameter changes:
+  - removed `externalSecrets.service.roleArn` (not required anymore with new SecretStores per Namespace)
+  - added `externalSecrets.service.secretstoreName` to set secretStore if required, default points to namespace secretStore generated by Crossplane Tenant XRD
+  - added `externalSecrets.service.refreshInterval` to control the refresh/sync interval from ExternalSecrets Operator. Default is set to 15 minutes.
+  The new Chart Version with new Secret Generator is only available after upgrade to ExternalSecret Version 0.7 (https://external-secrets.io/). It does not work with old Kubernetes External Secrets (https://github.com/external-secrets/kubernetes-external-secrets)
```

Comment:
Please highlight that with an indentation at the first place right after the version. It should be very obvious to users that this new version will break their current deployments.